.github/workflows/linting.yml

```diff
@@ -14,12 +12,9 @@ jobs:
         fetch-depth: 0
 
     - name: Vale
-      uses: errata-ai/vale-action@master
+      uses: errata-ai/vale-action@reviewdog
       with:
-        # We can modify these styles as we want
-        styles: |
-          https://github.com/errata-ai/Google/releases/latest/download/Google.zip
-        onlyAnnotateModifiedLines: true
+        filter_mode: nofilter
```

Comment:
By default, the `filter_mode` [is set to](https://github.com/errata-ai/vale-action#filter_mode-default-added) `added`. That ensures that reviewdog will only report on lines that have actually changed in a PR.

By specifying `nofilter`, Vale runs on the _entire_ codebase.

`added` should be good enough for now and mimics the `onlyAnnotateModifiedLines: true` option that we already had in the previous version of the Action. WDYT?

Reply:
I think that is fine - we should fix most of those findings in a separate PR (I can make a PR like I did for the end user docs) to bring it in line with expectations.